lib/workers/repository/dependency-dashboard.ts

```diff
@@ -47,6 +48,22 @@ function checkRebaseAll(issueBody: string): boolean {
   return issueBody.includes(' - [x] <!-- rebase-all-open-prs -->');
 }
 
+function checkCreateConfigMigrationPr(issueBody: string): string {
+  if (issueBody.includes('Config Migration necessary.')) {
```

Comment:
```suggestion
  if (issueBody.includes('Config Migration needed.')) {
```

Reply:
Yeah, this note maybe not be needed here.

We discussed in issue page to use a checkbox in the dashboard over the comment on PR.